packages/Bio/src/analysis/sequence-activity-cliffs.ts

```diff
@@ -164,7 +166,7 @@ export function createLinesGrid(df: DG.DataFrame, colNames: string[]): DG.Grid {
     .init((i) => `${df.get(colNames[0], i)}#${df.get(colNames[1], i)}`);
   seqDiffCol.semType = 'MacromoleculeDifference';
```

Comment:
why don't you export the existing `enum SEM_TYPES` and use `SEM_TYPES.MACROMOLECULE_DIFFERENCE`?